docs/source/development/technology-stack.rst

```diff
@@ -0,0 +1,150 @@
+(i.e. GitHub Issues) and progress (i.e. GitHub Projects), and performing code
+reviews. GitHub was chosen over other solutions like GitLab and Bitbucket mainly
+for its collaborative features and unlimited free public repositories. As the
+IMAP SDC strives to comply with the (relatively new) `NASA SMD SPD-41a policies
+<https://smd-cms.nasa.gov/wp-content/uploads/2023/08/smd-information-policy-spd-41a.pdf>`_,
+this open-source collaborative GitHub solution made the most sense.
+
+Poetry
+~~~~~~
+
+
+Pytest
+~~~~~~
+
+The `pytest <https://docs.pytest.org>`_ python library was chosen for writing
+and executing unit tests. This library was chosen because it has a large and
+active community of users, is well documented, and has several features that
+allow for a robust testing framework (e.g. fixtures, parametrized testing,
+object mocking, etc.). There are other tools/libraries for unit testing in
+Python (e.g. ``unittest``, ``nose``, etc.), but those were not really
+considered.
+
+
+Python
+~~~~~~
+
+The SDC uses Python as the primary programming language for the implementation
+of the vast majority of the system and its ancillary tools. A few other
+languages are supported for Level 3 `algorithm development containers
+<https://github.com/IMAP-Science-Operations-Center/imap_matlab_processing_example>`_,
+and a lot of the SDC cloud infrastructure is built with AWS CDK, but otherwise
+everything is written in Python. Python was chosen mostly because at the time of
+development, it was the most widely used and supported language for scientific
+software development, and the main programming language used within the
+Datasystems group at LASP.
+
+
+Space Packet Parser
+~~~~~~~~~~~~~~~~~~~
+
+The SDC uses the `space_packet_parser
+<https://space-packet-parser.readthedocs.io/en/stable/>`_ library for the
+decommutation of CCSDS packets and processing L0-level data. This library was
+chosen for its support of XTCE format for telemetry definitions. Another benefit
+to using this library is that it was developed and is actively maintained
+in-house here in Datasystems at LASP (Gavin Medley), and so the library can be
+updated to help meet the needs of the IMAP SDC. The other option for packet
+decommutation would have been `CCSDSPy <https://docs.ccsdspy.org/en/latest/>`_.
+
+Sphinx + ReadTheDocs
+~~~~~~~~~~~~~~~~~~~~
+
+The SDC uses the Python `sphinx <https://www.sphinx-doc.org/en/master/>`_
+library for generating project documentation as well as reference documentation
+that gets automatically rendered from code docstrings. `ReadTheDocs
+<https://about.readthedocs.com/?ref=readthedocs.com>`_ is used to host the
+documentation via the HTML files that ``sphinx`` generates. Together, these
+tools product the 'official' `IMAP SDC documentation
+<https://imap-processing.readthedocs.io>`_.
+
+These tools were chosen because they are widely used and have great integration
+with Poetry and GitHub.
+
+Sqlalchemy
+~~~~~~~~~~
+
+
+Xarray + cdflib
```

Comment:
oohh I see. Maybe disregard my previous comment, but move this up to just under the CDF section? If you title it "cdflib + Xarray" you can keep it alphabetical lol